webui/react/src/pages/ExperimentDetails.tsx

```diff
@@ -35,7 +37,7 @@ const ExperimentDetailsComp: React.FC = () => {
   }
```

Comment:
thought: I missed this on the last PR of this page, but it's possible for `experiment.error` to be triggered by a failure of decoding process, instead of the experiment actually failing to load. Maybe this should be a different PR, but my initial thought is to capture decoding errors in the `decode` function (under `ioTypes`) and throw a specific exception. Then in `apiConfig` we capture this exception and notify the user via notification when it occurs. This would be a pretty serious bug but if we see it, we will know exactly where and how to fix it.

Reply:
great idea we need this